src/ng/directive/ngRepeat.js

```diff
@@ -332,7 +332,7 @@
  *     For example: `item in items | filter:x as results` will store the fragment of the repeated items as `results`, but only after
  *     the items have been processed through the filter.
  *
- *     Please note that `as [variable name] is not an operator but rather a part of ngRepeat
```

Comment:
🔣🔣➖.     Please note that `as [variable name] is not an operator but rather a part of ngRepeat🔁🔣🔣